frontend/src/pages/footer.js

```diff
@@ -1,44 +1,44 @@
 import React from 'react';
-import {
-    Grid,
-    Typography
-} from '@material-ui/core';
-import COLORS from '../styles/colors'
+import { Grid, Typography } from '@material-ui/core';
+import COLORS from '../styles/colors';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles(theme => ({
-    root: {
-      flexGrow: 1,
-      backgroundColor: COLORS.grayLight,
-    }
-  }));
+  root: {
+    flexGrow: 1,
+    backgroundColor: COLORS.grayLight
+  }
+}));
 
 const Footer = props => {
-    const classes = useStyles();
+  const classes = useStyles();
 
-    return (
-        <Grid container className={classes.root} spacing={2}>
-            <Grid item xs={12}>
-                <Grid container justify="center" spacing={8}>
-                    <Grid item key="howItWorks">
-                        <Typography variant="h5">How it Works</Typography>
-                    </Grid>    
-                    <Grid item key="legal">
-                        <Typography variant="h5">Legal</Typography>
-                    </Grid>    
-                    <Grid item key="getInvolved">
-                        <Typography variant="h5">Get Involved</Typography>
-                    </Grid>
-                    
-                </Grid>
+  return (
+    <div style={{ paddingTop: '5rem' }}>
```

Comment:
As for the future development, the MUI has something called spacing (https://material-ui.com/customization/spacing/#spacing). It can be used to add spacing that are consistent with the overall theme. Feel free to check existing code how it's used, or reach out to us in Discord.

Reply:
Fixed it By adapting the documentation